### DISCUSSION TITLE
how to get github foundations certfications?

### DISCUSSION THREAD

**Discussion by Sameeratweb** [recodehive/Stackoverflow-Analysis#457] [Q&A] [answered]
i have already claimed student developer pack and got pro . i want to get certified  for free how I can ? i tried they are asking for any vouchers.

**Comment by SameerNadaf** [comment]
GitHub Foundations certification is part of GitHub’s training offerings, aimed at teaching the basics of GitHub and Git workflows. Unfortunately, certifications like GitHub Foundations are typically not free unless you have access to a voucher or special program.

Since you already have the **GitHub Student Developer Pack**, here's how you can explore certification opportunities for free or at a reduced cost:

---

### 1. **Check Student Developer Pack Benefits**
The Student Developer Pack often includes discounts or free access to educational platforms that offer certifications:
- Platforms like **GitHub Skills** (learn.github.com) offer free, interactive learning paths for Git and GitHub. While these don't directly provide certifications, they are great for skills building.
- Check the [Student Developer Pack page](https://education.github.com/pack) for updates or new additions related to certifications.

---

### 2. **Use GitHub Learning Lab (Free)**
GitHub offers **Learning Lab**, a free, hands-on platform where you can complete courses on GitHub-related topics:
- While these courses don’t provide certificates, they help you build skills that can be showcased in your portfolio.
- Visit: [GitHub Skills](https://skills.github.com/)

---

### 3. **Look for GitHub Events or Contests**
GitHub occasionally runs events, bootcamps, or contests where you can participate and get vouchers for free certifications. Keep an eye on:
- GitHub Education’s [Twitter](https://twitter.com/GitHubEducation) and [Blog](https://github.blog/education/).
- Local GitHub Campus Expert events at your college (if available).

---

### 4. **Ask Your Institution**
Many colleges or universities collaborate with GitHub Education. Reach out to your college professors or tech clubs to see if they can provide voucher codes for GitHub certifications.

---

### 5. **Look for Partner Discounts**
Some GitHub certifications are offered through partners like **Microsoft Learn**, which integrates GitHub into their training modules:
- Microsoft sometimes provides **free vouchers** for certifications like Azure or GitHub-related courses. Check for their student-focused campaigns or training days.

---

### 6. **Request a Voucher**
If vouchers are explicitly required, try reaching out:
- Contact **GitHub Support** through the Student Developer Pack support page and explain your interest.
- Email GitHub Education at `education@github.com` to inquire if they can provide guidance or access for students.

---

### 7. **Explore Free Certification Alternatives**
If GitHub-specific certifications remain inaccessible, consider free alternatives that cover GitHub and Git workflows:
- **freeCodeCamp**: Comprehensive Git and GitHub tutorials with certifications.
- **Codecademy**: Offers free Git tutorials (certifications might need Pro).
- **EdX or Coursera**: Look for free Git courses with certificates upon financial aid approval.

---

By exploring these options, you should be able to find free or low-cost certification opportunities to validate your GitHub skills.

**Reply by Sameeratweb** [reply]
Thankyou @SameerNadaf for guidance.

**Comment by techy4shri** [comment, marked as answer]
You will have a voucher code which comes with your student package. Checkout this video for step by step tutorial on how to get it and use it for free certifications: [Here](https://youtu.be/D7BQ_XiW384?si=7k8QcGhuzWRVDV5N)

**Comment by Aryanjstar** [comment]
right now it is free with student developer pack, so get it through their website by searching on google, claim it fast.

**Comment by techy4shri** [comment]
Hey @Sameeratweb do mark an answer which helped you here and feel free to ask further questions in case of doubt. Marking an answer is a great way of encouraging community support. Hope you have a good day :)

**Comment by MohamedAEleanany** [comment]
You will have a voucher code which comes with your student package.

**Reply by MohamedAEleanany** [reply]
a
